### DISCUSSION TITLE
테크놀로지 스택

### DISCUSSION THREAD

**Discussion by bryantson** [Korean-American-IT-Association-KAITA/kaita-project-management#45] [2. 자유토크]
자신있는 테크 스택 (예: HTML, Python, Java, SQL, 등등) 적어주세요

## 자신있는 분야 (1-5):
- Python:
- Java:
- HTML:
- CSS:
- React:

## 관심있는 분야 (1-5):
- Python:
- Java:
- HTML:
- CSS:
- React:

**Comment by ellijs** [comment]
Jessica Lee
-------------------
자신 있는 분야 
  - HTML(4), REACT(4), Javascript(4), CSS(3), SQL(3), Ruby(4) - 독학 및 붓캠프 과정 수료
------------------------------------------------------
관심 있는 분야
  - Python(1) (독학중), REACT(4)

**Reply by bryantson** [reply]
감사합니다. 곧 완성하겠습니다. @ellijs
